data/initrd/scripts/url/hmc/mount

```diff
@@ -0,0 +1,40 @@
+#! /bin/bash
+
+# https://github.com/openSUSE/linuxrc/blob/master/user_defined_url_schemes.txt
+
+set -e
+
+echo "url_dir = '$url_dir'"
+
+modprobe hmcdrv
+
+if [ -d "$url_dir" ] ; then
+  hmcdrvfs "$url_dir"
+
```

Comment:
I think we need to insert some checking in here to see if
1. The mount was successful
2. There is actually _something_ under the mountpoint
2a. It looks like an installation tree

Using lshmc, we can actually do 2 and 2a after the modprobe and _before_ the hmcdrvfs command